index.html

```diff
@@ -3598,13 +3587,13 @@ <h3>Evidence</h3>
 
         <p class="note">
 The `evidence` [=property=] provides different and complementary
-information to the `proof` [=property=]. The `evidence`
+information to the securing mechanism utlized. The `evidence`
 [=property=] is used to express supporting information, such as documentary
 evidence, related to the integrity of the [=verifiable credential=]. In
 contrast, the `proof` [=property=] is used to express
```

Comment:
Since the rest of this paragraph is comparing the `evidence` property to the securing mechanism, that should also happen here.
```suggestion
contrast, the securing mechanism is used to express
```